envoy/api/v2/cds.proto

```diff
@@ -51,7 +51,11 @@ message Cluster {
   string name = 1 [(validate.rules).string.min_bytes = 1];
 
   // An optional alternative to the cluster name to be used while emitting stats.
-  // Any ``:`` in the name will be converted to ``_`` when emitting statistics.
+  // Any ``:`` in the name will be converted to ``_`` when emitting statistics.This should not be
+  // confused with :ref:`Router Filter Header
+  // <config_http_filters_router_x-envoy-upstream-alt-stat-name>` which is a request header that
+  // emits statistics to a dual stat tree where as specifying **alt_stat_name** here emits
```

Comment:
I would terminate the sentence after `Router Filter Header` link and let the reader think about how they relate.